search/search-client-query-transformer/src/main/java/io/camunda/search/clients/transformers/ServiceTransformers.java

```diff
@@ -56,16 +56,17 @@
 import io.camunda.search.query.IncidentQuery;
 import io.camunda.search.query.ProcessDefinitionQuery;
 import io.camunda.search.query.ProcessInstanceQuery;
-import io.camunda.search.query.SearchQueryResult;
 import io.camunda.search.query.TypedSearchQuery;
 import io.camunda.search.query.UserQuery;
 import io.camunda.search.query.UserTaskQuery;
 import io.camunda.search.query.VariableQuery;
 import io.camunda.search.result.DecisionInstanceQueryResultConfig;
 import io.camunda.search.result.DecisionRequirementsQueryResultConfig;
 import io.camunda.search.result.ProcessInstanceQueryResultConfig;
+import io.camunda.search.sort.DecisionInstanceSort;
 import io.camunda.search.sort.FlowNodeInstanceSort;
 import io.camunda.search.sort.SortOption;
+import io.camunda.webapps.schema.entities.operate.dmn.DecisionInstanceEntity;
```

Comment:
🔧 Ideally, we don't have component-specific stuff in here as we want to move away from components, so anything containing `Operate` or such should be avoided.

❓ Are the transformers only used by ES/OS but not RDBMS? Just trying to understand on which level we're introducing those specific entities now.

Reply:
>Ideally, we don't have component-specific stuff in here as we want to move away from components, so anything containing Operate or such should be avoided.
This is the datamodel that is used by the new exporter `io.camunda:webapps-schema` :/ it seems they kept, for some reason, `operate` in the package names

>❓ Are the transformers only used by ES/OS but not RDBMS? Just trying to understand on which level we're introducing those specific entities now.
Yes these transformers are exclusive to ES/OS. RDBMS has other database model and uses other transformers (see `RdbmsSearchClient`)

Reply:
> This is the datamodel that is used by the new exporter io.camunda:webapps-schema :/ it seems they kept, for some reason, operate in the package names

Understood, not our "fault" then, thanks for clarifying 👍 We should flag this with the exporter team though in my view. @houssain-barouni, can you ask them why they kept the component name?

Reply:
it seems this will be cleaned soon https://camunda.slack.com/archives/C06F0GLJNFM/p1731667183101189?thread_ts=1731666811.834889&cid=C06F0GLJNFM